tests/cmsis_rtos_v1/src/semaphore.c

```diff
@@ -16,6 +16,10 @@ void thread_sema(void const *arg)
 {
 	int tokens_available;
 
+	/* Try taking when no semaphore is available */
+	zassert_false(osSemaphoreWait((osSemaphoreId)arg, 0),
+			"Something's wrong with osSemaphoreWait!");
+
```

Comment:
Please modify the statement above to make it similar to the below statements. Also, change the comment to indicate this is a testcase with 0 timeout.